_posts/2019-01-29-OAS-3.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenAPI 3.0
+layout: post
+date: 2019-01-29 15:00:00 +0000
+author: kyle
+published: true
+comments: true
+---
+
+I'm delighted to announce experimental support for OpenAPI 3.0 in Apiary.
+OpenAPI 3 is the new version of the OpenAPI Specification, the successor to
+OpenAPI 2 (or formerly known as Swagger 2).
+
+< FIXME Add screenshot of OAS 3 doc in Apiary using Apiary UI >
```

Comment:
Preparing a nice OpenAPI 3 example and will make a screenshot to go here before going live.